recipes/salib/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.0.1" %}
+
+package:
+  name: salib
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/SALib/SALib
+  git_tag: v{{ version }}
```

Comment:
that's what I went for at first, but they use pyscaffold to manage the build and it wants to be in a git repo to build, else it errors.